### DISCUSSION TITLE
Pre-Proposal Form

### DISCUSSION THREAD

**Discussion by ummyhajj** [MDAnalysis/mdanalysis#4500] [Outreachy Discussions]
I am currently working on the pre-proposal. It has to be submitted not later than March 22, 2024.

**Comment by ummyhajj** [comment]
Take your time and research to give a very good proposal👍

**Comment by ummyhajj** [comment]
How's is going with the pre-proposal?

**Comment by ummyhajj** [comment]
I have submitted my pre-poposal form. Where can I access the final application form?

**Reply by tareema** [reply]
Hello! Congratulations on submitting your pre-proposal early. According to one of the mentors, they would reach out to us and let us know if we can proceed with submitting the final application after they go through the preproposal submissions. I hope I was able to answer your question

**Reply by ummyhajj** [reply]
Yes! Thank you.
